cmake/podioConfig.cmake.in

```diff
@@ -21,6 +21,7 @@
 set_and_check(podio_INCLUDE_DIR "@PACKAGE_CMAKE_INSTALL_INCLUDEDIR@")
 set_and_check(podio_LIBRARY_DIR "@PACKAGE_CMAKE_INSTALL_LIBDIR@")
 set_and_check(podio_PYTHON_DIR "@PACKAGE_podio_PYTHON_INSTALLDIR@")
+set(podio_IO_HANDLERS "@PODIO_IO_HANDLERS@")
```

Comment:
I was aiming for consistency with the other exported variables, where `podio` is used in lower case ;)
But it would indeed be more consistent in upper case, also with regards to the scheme for the macros.

Reply:
Changed, and also updated the release notes.